dashboard/app/controllers/foorm_form_editor_controller.rb

```diff
@@ -1,10 +1,10 @@
 # Foorm Editor is only available on levelbuilder or test, for those with levelbuilder permissions.
-class FoormEditorController < ApplicationController
+class FoormFormEditorController < ApplicationController
```

Comment:
could we do instead Foorm::FormEditorController (similarly Foorm::LibraryEditorController)? That would follow the patter of Foorm::Form and Foorm::LibraryQuestion and be more readable

Reply:
Makes sense to me!